keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -407,6 +415,24 @@ marked UnschedulableAndUnresolvable.
 [volume restrictions plugin]: https://github.com/kubernetes/kubernetes/blob/v1.21.0/pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go#L29
 [node info cache]: https://github.com/kubernetes/kubernetes/blob/v1.21.0/pkg/scheduler/framework/types.go#L357
 
+##### Beta
+
+Support for pod preemption is enforced in beta.
+
+When a pod (A) with a ReadWriteOncePod PVC is scheduled, if another pod (B) is
+found using the same PVC and pod (A) has higher priority, the scheduler will
+return an "Unschedulable" status and pod (B) will be preempted.
```

Comment:
the filter plugin will return Unschedulable status, triggering the preemption logic to attempt to preempt pod (B).


There are no guarantees that the pod would be preempted, for example, if B is too small, A might not fit.